cmd/operator-sdk/new/cmd.go

```diff
@@ -382,8 +384,16 @@ func getDeps() error {
 		if err := execProjCmd("dep", "ensure", "-v"); err != nil {
 			return err
 		}
+	case projutil.DepManagerGoMod:
+		log.Info("Running go mod ...")
```

Comment:
Nit: 
```suggestion
		log.Info("Running `go mod` ...")
```

Reply:
I don't think this is necessary, and none of the other commands have backticks.